### ISSUE TITLE
ValueError: Error when checking : expected input_1 to have shape (None, None, 4) but got array with shape (1, 12, 3)

### ISSUE THREAD

**Issue by sradiouy** [nofundamental/lncRNAnet#2]
Hi, i am having problems running lncRNAnet. I get the following issue:

ValueError: Error when checking : expected input_1 to have shape (None, None, 4) but got array with shape (1, 12, 3)

Any clues? 

Im a using the requirements file to install the programs.

**Comment by florekem** [comment]
Try with smaller input file. I have similar error when running with larger 'batches'.

**Comment by pentalpha** [comment]
> Try with smaller input file. I have similar error when running with larger 'batches'.

Indeed, I have this exact error when using a 43M '.fasta' file, but the error does not happen with a 24M file.
